actors/builtin/market/policy.go

```diff
@@ -20,11 +23,21 @@ func dealPricePerEpochBounds(size abi.PaddedPieceSize, duration abi.ChainEpoch)
 	return abi.NewTokenAmount(0), abi.TotalFilecoin // PARAM_FINISH
 }
 
-func dealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, duration abi.ChainEpoch) (min abi.TokenAmount, max abi.TokenAmount) {
-	return abi.NewTokenAmount(0), abi.TotalFilecoin // PARAM_FINISH
+func DealProviderCollateralBounds(pieceSize abi.PaddedPieceSize, verified bool, networkQAPower, baselinePower abi.StoragePower, networkCirculatingSupply abi.TokenAmount) (min abi.TokenAmount, max abi.TokenAmount) {
+	lockTargetNum := big.Mul(ProvCollateralPercentSupplyNum, networkCirculatingSupply)
```

Comment:
It would nice to have a succinct math formula as a comment here.

Reply:
great idea will do